src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+export type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+export interface Figure {
+  color: Color;
+  shape: Shape;
+  getArea(): number;
+}
+
```

Comment:
Recommend creating a separate function like this:
```
function checkSides(errorMessage: string, ...sides: number[]): void {
  if (sides.some(side => side <= 0)) {
    throw new Error(errorMessage);
  }
}
```
because this check is used three times